<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nintendo DS Banner Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="5">
     <widget class="QGroupBox" name="gfx_gb">
      <property name="title">
       <string>Graphics</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="4" column="0">
        <widget class="QPushButton" name="gfxImport_pb">
         <property name="text">
          <string>Import Image</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="gfxExport_pb">
         <property name="text">
          <string>Export Image</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="IconGraphicsView" name="gfx_gv"/>
       </item>
       <item row="0" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="gfxBmp_label">
           <property name="text">
            <string>Bitmap:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="gfxBmp_sb">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>8</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="gfxPal_label">
           <property name="text">
            <string>Palette:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="gfxPal_sb">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>8</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="1" rowspan="2">
     <widget class="QGroupBox" name="anim_gb">
      <property name="title">
       <string>Animation</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="3" column="2" colspan="4">
        <widget class="QSpinBox" name="animPal_sb">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>8</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="2">
        <widget class="QComboBox" name="animFrame_cb">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QLabel" name="animFlip_label">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Flip:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="animBmp_label">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Bitmap:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="animFrameAdd_pb">
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" colspan="4">
        <widget class="QSpinBox" name="animBmp_sb">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>8</number>
         </property>
        </widget>
       </item>
       <item row="1" column="2" colspan="4">
        <widget class="QSpinBox" name="animDur_sb">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="animFrame_label">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Frame:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="animDur_label">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Duration:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLabel" name="animPal_label">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Palette:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QCheckBox" name="animFlipX_cb">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="animFrameRem_pb">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QCheckBox" name="animFlipY_cb">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QGroupBox" name="gameTitle_gb">
      <property name="title">
       <string>Title</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QComboBox" name="gameTitle_cb">
         <item>
          <property name="text">
           <string>Japanese</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>English</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>French</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>German</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Italian</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Spanish</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Chinese</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Korean</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QPlainTextEdit" name="gameTitle_pte"/>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="gameTitle_pb">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Set All</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QGroupBox" name="bannerVersion_gb">
      <property name="title">
       <string>Version</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QComboBox" name="bannerVersion_cb">
         <item>
          <property name="text">
           <string>0x0001 – Regular DS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0x0002 – Chinese DS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0x0003 – Korean DS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0x0103 – DSi</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionQt"/>
    <addaction name="actionCredits"/>
   </widget>
   <widget class="QMenu" name="menuExtra">
    <property name="title">
     <string>Extra</string>
    </property>
    <addaction name="actionAnimation_Player"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="actionEnglish"/>
    <addaction name="actionJapanese"/>
    <addaction name="actionPortugu_s"/>
    <addaction name="actionFrench"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuExtra"/>
   <addaction name="menuLanguage"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+N</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+W</string>
   </property>
  </action>
  <action name="actionCredits">
   <property name="text">
    <string>&amp;This</string>
   </property>
  </action>
  <action name="actionQt">
   <property name="text">
    <string>&amp;Qt</string>
   </property>
  </action>
  <action name="actionAnimation_Player">
   <property name="text">
    <string>&amp;Animation Player</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+P</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string notr="true">English</string>
   </property>
  </action>
  <action name="actionJapanese">
   <property name="text">
    <string notr="true">日本語</string>
   </property>
  </action>
  <action name="actionPortugu_s">
   <property name="text">
    <string notr="true">Português</string>
   </property>
  </action>
  <action name="actionFrench">
   <property name="text">
    <string notr="true">Français</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IconGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>mainwindow.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
